sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -71,6 +71,7 @@ final class ManifestMetadataReader {
 
   static final String ATTACH_SCREENSHOT = "io.sentry.attach-screenshot";
   static final String CLIENT_REPORTS_ENABLE = "io.sentry.send-client-reports";
+  static final String COLLECT_IPC_DEVICE_INFO = "io.sentry.collect-ipc-device-info";
```

Comment:
```suggestion
  static final String COLLECT_IPC_DEVICE_INFO = "io.sentry.ipc-device-context";
```

I'd rather rename the flags to be named -> IPC Device Context.
Otherwise, everything would every collect as a prefix.